customers/tests/conftest.py

```diff
@@ -1,6 +1,7 @@
 import pytest
 
 from berth_reservations.tests.conftest import *  # noqa
+from resources.tests.conftest import boat_type  # noqa
```

Comment:
I think `# noqa` is not needed, since the line above has it, because default isort config complains about wildcard imports

Reply:
It is, because of the `unused import` error.

Reply:
ah, right-right-right! 👍